components/playlist/playlist_download_request_manager.cc

```diff
@@ -162,6 +163,7 @@ void PlaylistDownloadRequestManager::GetMedia(content::WebContents* contents) {
 void PlaylistDownloadRequestManager::OnGetMedia(
     base::WeakPtr<content::WebContents> contents,
     base::Value value) {
+  VLOG(2) << __func__;
```

Comment:
we need to cleanup this after some time or use `DVLOG`, otherwise we just increase binary size for no good reason.

Reply:
Thanks! done.